cmd/manager/main.go

```diff
@@ -28,7 +28,7 @@ import (
 	// to ensure that exec-entrypoint and run can make use of them.
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
-	api "github.com/openebs/jiva-operator/pkg/apis/openebs/v1"
+	jivaapi "github.com/openebs/jiva-operator/pkg/apis/openebs/v1"
```

Comment:
Can we use camel case for the naming like jivaAPI instead of jivaapi

Reply:
Updated. Rest of the codebase uses all small cases though.